File: Assignment_33/Program2.java
import java.util.*;

class ChkAlpha
{
    public boolean isAlphabet(char ch)
    {
        if(ch >= 'A' && ch <= 'Z')
        {
            return true;
        }
        else
        {
            return false;
        }   
    }     
}
public class Program2
{
    public static void main(String[] args)
    {
    
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter the Character:");
        char ch = sobj.next().charAt(0);

        ChkAlpha cobj = new ChkAlpha();
        boolean bRet = cobj.isAlphabet(ch);

        if(bRet)
        {
            System.out.println("TRUE");
        }
        else
        {
            System.out.println("FALSE");
        }

    }
}

File: Assignment_33/Program2.c
#include<stdio.h>

#define TRUE 1
#define FALSE 0
typedef int BOOL;

BOOL ChkAlpha(char *ch)
{
        if(*ch >= 'A' && *ch <= 'Z')
        {
            return TRUE;
        }
        else
        {
            return FALSE;
        }     
}
int main()
{
     char Arr[50] = {'\0'};
     BOOL bRet = FALSE; 

     printf("Enter Character :\n");
     scanf("%[^'\n']c",Arr);

     bRet = ChkAlpha(Arr);

     if(bRet == TRUE)
     {
        printf("TURE\n");
     }
     else
     {
        printf("FALSE\n");
     }
     return 0;
}